kolla/defaults.yml

```diff
@@ -108,6 +108,8 @@ glance_backends:
     enabled: "{{ glance_backend_swift | bool }}"
 
 glance_allow_anonymous_access: no
+glance_property_protections_file: "{{ node_custom_config }}/glance/property_protections.conf"
```

Comment:
is this the correct way to make sure we get this property_protections.conf file propagated where we need it?